integreat_cms/cms/models/regions/region.py

```diff
@@ -417,6 +417,7 @@ class Region(AbstractBaseModel):
         max_length=256,
         blank=True,
         default="",
+        unique=True,
```

Comment:
We cannot allow to regions to use the same Zammad server, because we need the reverse look up of the Zammad server IP during webhooks to get the region to which the Zammad belongs. This is also a good idea for privacy reasons.